examples/full.py

```diff
@@ -3,178 +3,230 @@
+		"type": "actions",
+		"elements": [
+			{
+				"type": "button",
+				"text": {
+					"type": "plain_text",
+					"emoji": True,
+					"text": "Yes"
+					},
+				"style": "primary",
+				"value": "i_like_bots",
+				"action_id": YES
+				},
+			{
+				"type": "button",
+				"text": {
+					"type": "plain_text",
+					"emoji": True,
+					"text": "No"
+					},
+				"style": "danger",
+				"value": "i_dont_like_bots",
+				"action_id": NO
+				}
+			]
+		}
+	blocks = [
+		text_block('Do you like Bots?'),
+		yes_no_buttons_block
+		]
+	return block_reply(blocks)
 
 
 @slackify.action("yes")
 def yes(action):
-    text_blok = text_block('Super! I do too :thumbsup:')
-    respond(action['response_url'], {'blocks': [text_blok]})
-    return OK
+	text_blok = text_block('Super! I do too :thumbsup:')
+	respond(action['response_url'], {'blocks': [text_blok]})
+	return OK
 
 
 @slackify.action("no")
 def no(action):
-    text_blok = text_block('Boo! You are so boring :thumbsdown:')
-    respond(action['response_url'], {'blocks': [text_blok]})
-    return OK
+	text_blok = text_block('Boo! You are so boring :thumbsdown:')
+	respond(action['response_url'], {'blocks': [text_blok]})
+	return OK
 
 
 @slackify.command
 def register():
-    username_input_block = {
-        "type": "input",
-        "block_id": "username_block",
-        "element": {
-            "type": "plain_text_input",
-            "placeholder": {
-                "type": "plain_text",
-                "text": "Enter your username"
-            },
-            "action_id": "username_value"
-        },
-        "label": {
-            "type": "plain_text",
-            "text": "👤 Username",
-            "emoji": True
-        }
-    }
-    password_input_block = {
-        "type": "input",
-        "block_id": "password_block",
-        "element": {
-            "type": "plain_text_input",
-            "placeholder": {
-                "type": "plain_text",
-                "text": "Enter your password"
-            },
-            "action_id": "password_value"
-        },
-        "label": {
-            "type": "plain_text",
-            "text": "🔑 Password",
-            "emoji": True
-        }
-    }
-    modal_blocks = [
-        username_input_block,
-        password_input_block,
-    ]
-    callback_id = 'registration_form'
-    registration_form = {
-        "type": "modal",
-        "callback_id": callback_id,
-        "title": {
-            "type": "plain_text",
-            "text": "My First Modal",
-            "emoji": True
-        },
-        "submit": {
-            "type": "plain_text",
-            "text": "Register",
-            "emoji": True
-        },
-        "close": {
-            "type": "plain_text",
-            "text": "Cancel",
-            "emoji": True
-        },
-        "blocks": modal_blocks
-    }
-    cli.views_open(
-        trigger_id=request.form['trigger_id'],
-        view=registration_form
-    )
-    return OK
+	username_input_block = {
```

Comment:
I think you have configured spacing with tabs, the repo uses 4 spaces instead. Please review changes and revert them so we keep consistency across all files

Perhaps i should add a linter check to avoid this :sweat_smile:

Reply:
I have a better idea. I add a linter check so I catch it before pushing it next time? 😂